plugins/slack_app/app/controllers/slack_app_controller.rb

```diff
@@ -0,0 +1,158 @@
+class SlackAppController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: [:command, :interact]
+  skip_around_action :login_user, except: :oauth
+
+  before_action :handle_slack_inputs
+  before_action :verify_environment, except: [:oauth]
```

Comment:
Don't listen to that other panda ...

On Mon, Jul 18, 2016 at 10:58 AM, Ben Straub <notifications@github.com>
wrote:

> In plugins/slack_app/app/controllers/slack_app_controller.rb
> <https://github.com/zendesk/samson/pull/1130#discussion_r71196418>:
>
> > @@ -0,0 +1,158 @@
> > +class SlackAppController < ApplicationController
> > +  skip_before_action :verify_authenticity_token, only: [:command, :interact]
> > +  skip_around_action :login_user, except: :oauth
> > +
> > +  before_action :handle_slack_inputs
> > +  before_action :verify_environment, except: [:oauth]
>
> Ha. You suggested the opposite earlier. 😀 I'll fix.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1130/files/72880ddde5474136f82a39966614c4efa3618fa7#r71196418>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ0hPvVGM3YMf6uOFFraxBbKCTQmiks5qW77WgaJpZM4JIRHk>
> .
>